<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>insertDialog</class>
 <widget class="QDialog" name="insertDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>插入</string>
  </property>
  <property name="windowIcon">
   <iconset resource="recourse.qrc">
    <normaloff>:/new/prefix1/insert.png</normaloff>:/new/prefix1/insert.png</iconset>
  </property>
  <widget class="QTabWidget" name="studypage">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>601</width>
     <height>441</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;study&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;宋体&quot;;</string>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>class</string>
    </attribute>
    <widget class="QPushButton" name="insert_class_Button">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>350</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>插入</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancel">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>350</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>381</width>
       <height>241</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>班号：</string>
        </property>
        <property name="buddy">
         <cstring>class_cno_Edit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="class_cname_Edit"/>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="class_cno_Edit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>班级名：</string>
        </property>
        <property name="buddy">
         <cstring>class_cname_Edit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>专业：</string>
        </property>
        <property name="buddy">
         <cstring>class_major_Edit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="class_major_Edit"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="StudentInfo">
    <attribute name="title">
     <string>StudentInfo</string>
    </attribute>
    <widget class="QPushButton" name="insert_stuinfo_Button">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>350</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>插入</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancel_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>350</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>431</width>
       <height>291</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="styleSheet">
         <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
        </property>
        <property name="text">
         <string>学号：</string>
        </property>
        <property name="buddy">
         <cstring>stuinfo_sno_Edit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="stuinfo_sno_Edit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>姓名：</string>
        </property>
        <property name="buddy">
         <cstring>stuinfo_sname_Edit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="stuinfo_sname_Edit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>性别：</string>
        </property>
        <property name="buddy">
         <cstring>stuinfo_ssex_Edit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="stuinfo_ssex_Edit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>入学年份：</string>
        </property>
        <property name="buddy">
         <cstring>stuinfo_entry_Edit</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_9">
        <property name="text">
         <string>班号：</string>
        </property>
        <property name="buddy">
         <cstring>stuinfo_classnum_Edit</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="stuinfo_classnum_Edit"/>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="stuinfo_entry_Edit"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>course</string>
    </attribute>
    <widget class="QPushButton" name="insert_course_Button">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>350</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>插入</string>
     </property>
    </widget>
    <widget class="QPushButton" name="cancel_3">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>350</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>441</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>课程号：</string>
        </property>
        <property name="buddy">
         <cstring>course_cno_Edit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="course_cno_Edit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>课程名：</string>
        </property>
        <property name="buddy">
         <cstring>course_cname_Edit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="course_cname_Edit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>学分：</string>
        </property>
        <property name="buddy">
         <cstring>course_credit_Edit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="course_credit_Edit"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>study</string>
    </attribute>
    <widget class="QPushButton" name="cancel_4">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>350</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
    <widget class="QPushButton" name="insert_study_Button">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>350</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 12pt &quot;微软雅黑&quot;;</string>
     </property>
     <property name="text">
      <string>插入</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>421</width>
       <height>221</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>学号：</string>
        </property>
        <property name="buddy">
         <cstring>study_sno_Edit</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="study_sno_Edit"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>课程号：</string>
        </property>
        <property name="buddy">
         <cstring>study_cno_Edit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="study_cno_Edit"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>成绩：</string>
        </property>
        <property name="buddy">
         <cstring>study_grade_Edit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="study_grade_Edit"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="recourse.qrc"/>
 </resources>
 <connections/>
</ui>
